js/evidence.js

```diff
@@ -1,26 +1,196 @@
-				$(this).css('background-image', 'url(' + mobile_background_image + ')' );
-			} else if ( background_image ) {
-				$(this).css('background-image', 'url(' + background_image + ')' );
+			if (990 > $(window).width() && mobile_background_image) {
+				$(this).css('background-image', 'url(' + mobile_background_image + ')');
+			} else if (background_image) {
+				$(this).css('background-image', 'url(' + background_image + ')');
 			}
 		});
-	};
+	}
+
+	/**
+	 * Apply `fixed` class to the filter drop-down if it has been scrolled to/past.
+	 */
+	function fix_filters() {
+		var $wrapper_offset = $wrapper.offset().top - offset;
+
+		$(document).on('scroll', function () {
+			if ($(window).scrollTop() >= $wrapper_offset) {
+				$wrapper.addClass('fixed');
+			} else {
+				$wrapper.removeClass('fixed');
+			}
+		});
+	}
+
+	/**
+	 * Check if the footer is in the viewport.
+	 */
+	function footer_is_visible() {
+		var footer = $('.site-footer');
+
+		if (footer instanceof jQuery) {
+			footer = footer[0];
+		}
+
+		var rect = footer.getBoundingClientRect();
+
+		return rect.bottom > 0 &&
+			rect.right > 0 &&
+			rect.left < (window.innerWidth || document.documentElement.clientWidth) &&
+			rect.top < (window.innerHeight || document.documentElement.clientHeight);
+	}
+
+	/**
+	 * Make an AJAX call and display the response.
+	 */
+	function fetch_and_display_stories(data, type) {
+		$.ajax({
+			type: 'post',
+			url: evidence.ajax_url,
+			data: data,
+			beforeSend: function () {
+				fetching = true;
+				if ('scrolled' === type) {
+					$story_container.append('<div class="loading"></div>');
+				} else if ('filtered' === type) {
+					$story_container.find('.section-wrapper').hide(400);
+				}
+			}
+		}).done(function (response) {
+			if ('scrolled' === type) {
+				$('.loading').remove();
+				$story_container.append($.parseJSON(response));
+				process_column_backgrounds();
+				$story_container.data('page', data.page + 1);
+			} else if ('filtered' === type) {
+				$story_container.html($.parseJSON(response));
+				process_column_backgrounds();
+				$story_container.data('page', 1);
+			}
+			$('.new').show(400);
+		}).always(function () {
+			fetching = false;
+		});
+	}
+
+	/**
+	 * Load more posts when the footer is reached.
+	 */
+	function infinite_scroll() {
+		window.addEventListener('scroll', function () {
+			if (fetching ||
+					all_stories ||
+					!footer_is_visible() ||
+					$story_container.data('page') === $story_container.data('total-pages') ||
+					'' !== $('#filter-options').find('option:selected').val()) {
+				return;
+			}
+
+			var data = {
+					action: 'evidence_stories',
+					nonce: evidence.nonce,
+					page: $story_container.data('page')
+				};
+
+			fetch_and_display_stories(data, 'scrolled');
+		}, 500);
+	}
+
+	/**
+	 * Scroll to the filter wrapper when an option is selected.
+	 */
+	function scroll_to_filter() {
+		var filters_top = Math.round($filters.offset().top - offset);
```

Comment:
It may be possible to capture this when the script first initializes. Though, if this isn't hit that often, it may not matter much.